build.linux32x86/third-party/libgit2/Makefile.lib

```diff
@@ -0,0 +1,52 @@
+include ../common/Makefile.lib.extra
+
+$(info LIBRARY: libgit2)
+
+# plugin definitions
+LIBGIT2URL:=https://github.com/libgit2/libgit2/archive/v0.24.2.tar.gz
+LIBGIT2DIR:=$(THIRDPARTYDIR)/libgit2-0.24.2
+LIBGIT2LIBNAME:=libgit2.so.0.24.2
+LIBGIT2LIB:=$(THIRDPARTYINSTALLDIR)/$(LIBGIT2LIBNAME)
+LIBGIT2ARCHIVE:=$(THIRDPARTYCACHEDIR)/libgit2-v0.24.2.tar.gz
+
+# ensure third-party library is built and recognised by plugins
+INCDIRS:=$(INCDIRS) $(THIRDPARTYINCLUDEDIR)
+EXTRALIBS:=$(EXTRALIBS) $(LIBGIT2LIB)
+PLUGINREQS:=$(THIRDPARTYLIBS)
+
+$(LIBGIT2ARCHIVE): 
+	curl -o $(LIBGIT2ARCHIVE) -L $(LIBGIT2URL) 
+
+$(LIBGIT2DIR)/CMakeLists.txt: $(LIBGIT2ARCHIVE)
+	tar x -f $(LIBGIT2ARCHIVE) -C $(THIRDPARTYDIR)
+	touch $(LIBGIT2DIR)/CMakeLists.txt
+
+# NOTE: I'm adding this ugly hack of "sed..." because CMake does not find properly libssl and libcrypto 
+# on 64bits envirenments (it always find the 64bits version, regardless the architecture).
+# this is ugly, but the only way I fond :( 
```

Comment:
Did you try https://cmake.org/cmake/help/v3.0/prop_gbl/FIND_LIBRARY_USE_LIB64_PATHS.html ?